README.md

```diff
@@ -27,115 +31,167 @@ repositories' pull requests by using the GitHub Pull Request Builder plugin. It
 will start a batch of tests whenever a new pull request is open in the "builds"
 and "versions" repositories by an authorized user, when those pull requests are
 updated with new commits or when an authorized user comments the phrase
-"start tests". Builds and Pylint validation can be triggered independently with
-the phrases "start build" and "start pylint", respectively.
-
-### Release process
-
-A Jenkins job will trigger periodically to update the packages' versions, create
-release notes and execute a build with all packages available, by executing the
-corresponding commands from the "builds" repository scripts. Commits will be
-created for the "versions" and "open-power-host-os.github.io" repositories and
-pushed to the designated GitHub user's repository. Ideally, this should be a
-"bot" user. Pull requests for those commits can be created manually and merged
-into the main organization/user repository, if desired.
-
-Note: The default is to run those periodic builds once a week (every Wednesday
-at 11 AM).
+"start tests". Currently, tests are: building of all packages, Pylint validation
+of Python code, yamllint validation of all package metadata YAML files and rpmlint
+validation of all RPM packages specification files. These tests can be triggered
+independently with the phrases "start build", "start pylint", "start yamllint"
+and "start rpmlint" respectively.
+
+### Periodic builds process
+
+A Jenkins job will trigger periodically to check the versions of the packages
+available in OpenPOWER Host OS GitHub organization, update the corresponding
+packages' versions in the "versions" git repository, create a build with all
+packages available and create release notes, by executing the corresponding
+commands from the build scripts in the "builds" git repository. Commits will
+be created for the local "versions" and "open-power-host-os.github.io" git
+repositories and pushed to the designated GitHub user's git repository. Ideally,
+this should be a "bot" user. Pull requests for those commits can be created
+manually and merged into the main organization/user git repository, if desired.
+The default is to execute those periodic builds once a week (every Wednesday
+at 11 AM, Jenkins master's timezone).
+
+There's another periodic job similar to the one above, which differs by executing
+additional testing during the packages builds and by not pushing commits to remote
+git repositories. It allows developers of the packages to have their new commits
+tested and to have a build with those commits as soon as possible. Note: The default
+is to execute those periodic builds nightly (daily at 22 PM, Jenkins master's timezone).
```

Comment:
Daily means "every day", not "during the day", and refers to a periodicity as weekly, monthly and yearly do. Nightly does not refer to periodicity, but only to the period of the day when it happens. IMHO daily is more accurate than nightly here.